lib/scidata/yelp_polarity_reviews.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Scidata.YelpPolarityReviews do
+  @moduledoc """
+  Module for downloading the [Yelp Reviews dataset](https://www.yelp.com/dataset).
```

Comment:
```suggestion
  Module for downloading the [Yelp Polarity Reviews dataset](https://course.fast.ai/datasets#nlp).
```

Let's link to the Fast AI NLP datasets here, as there doesn't seem to be any information on the polarity dataset on the Yelp page. But please let me know if it does exist on the latter and I've missed it!

Reply:
Good idea! It looks like both [Huggingface Datasets](https://huggingface.co/datasets/yelp_polarity) and [Tensorflow Datasets](https://www.tensorflow.org/datasets/catalog/yelp_polarity_reviews) are also using the data from Fast AI.